src/views/rewards/rewards.vue

```diff
@@ -4,9 +4,31 @@
 
 <template>
   <div>
-    <Header :showClose=true />
+    <Header :showClose="true" />
     <div class="rewards">
-      <h1 class="rewards__header">This is the rewards page</h1>
+      <h1 class="header-2">Your Rewards</h1>
+      <p class="rewards-email">{{email}}</p>
+      <ProgressBannerComponent />
+      <RewardDetails
+        :title="'Visit 1'"
+        :subTitle="'One entry to win'"
+        :unLocked="this.visitedVenues.length ? true : false"
+      />
+      <RewardDetails
+        :title="'Visit 3'"
```

Comment:
string literals can just be passed in like this:

```
title="Visit 1"
```
instead of using the colon and double quotes

Reply:
Nice! Just made the change.